service/version_bundles.go

```diff
@@ -227,5 +227,46 @@ func newVersionBundles() []versionbundle.Bundle {
 			Version:      "2.0.0",
 			WIP:          false,
 		},
+		{
+			Changelogs: []versionbundle.Changelog{
+				{
+					Component:   "cloudconfig",
+					Description: "Fix eventual decryption race.",
+					Kind:        versionbundle.KindFixed,
+				},
+			},
+			Components: []versionbundle.Component{
+				{
+					Name:    "calico",
+					Version: "3.0.1",
+				},
+				{
+					Name:    "docker",
+					Version: "17.09.0",
+				},
+				{
+					Name:    "etcd",
+					Version: "3.2.7",
+				},
+				{
+					Name:    "coredns",
+					Version: "1.0.5",
+				},
+				{
+					Name:    "kubernetes",
+					Version: "1.9.2",
+				},
+				{
+					Name:    "nginx-ingress-controller",
+					Version: "0.10.2",
+				},
+			},
+			Dependencies: []versionbundle.Dependency{},
+			Deprecated:   false,
+			Name:         "aws-operator",
+			Time:         time.Date(2018, time.January, 30, 11, 55, 0, 0, time.UTC),
+			Version:      "2.0.1",
+			WIP:          true,
```

Comment:
We should deprecate the former version bundle so customer are not able to use the version without the fix. We should then make this WIP == false so the customer has a release to create clusters for.

Reply:
+1

Reply:
Done, PTAL.